src/CommunityToolkit.Maui.Core/Essentials/FileSaver/FileSaverImplementation.android.cs

```diff
@@ -113,6 +113,7 @@ static async Task<string> SaveDocument(AndroidUri uri, Stream stream, Cancellati
 			ArrayPool<byte>.Shared.Return(buffer);
 		}
 
+		parcelFileDescriptor?.Close();
```

Comment:
Why do we need to call it explicitly? ParcelFileDescriptor is disposed at the end.

Reply:
When I was debugging #1588 using "CommunityToolkit' source code, I found an issue with Android 7.1 and did not confirm other versions of Android.
It can be confirmed that Android 7.1 must call `Close()`